File: app/src/main/java/xyz/klinker/messenger/adapter/view_holder/AttachContactViewHolder.java
package xyz.klinker.messenger.adapter.view_holder;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import xyz.klinker.messenger.R;

public class AttachContactViewHolder extends RecyclerView.ViewHolder {

    public ImageView picture;
    public TextView name;

    public AttachContactViewHolder(View itemView) {
        super(itemView);

        name = (TextView) itemView.findViewById(R.id.name);
        picture = (ImageView) itemView.findViewById(R.id.picture);
    }

}

File: app/src/main/java/xyz/klinker/messenger/adapter/view_holder/AttachContactViewHolder.kt
package xyz.klinker.messenger.adapter.view_holder

import android.view.View
import android.widget.ImageView
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView

import xyz.klinker.messenger.R

class AttachContactViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {

    val picture: ImageView = itemView.findViewById<View>(R.id.picture) as ImageView
    val name: TextView = itemView.findViewById<View>(R.id.name) as TextView
}
